### PR TITLE
[ADD] ecommerce: new page on pricing management

### PR DESCRIPTION
**Pull request by toaa-odoo**
Created a new page describing how to manage pricelists, discounts, displayed prices, etc. V14 only.

taskid: 3069487

### CODE REVIEW

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 1]
```diff
@@ -0,0 +1,185 @@
+==================
+Pricing management
+==================
+
+Odoo offers multiple options to select the price displayed on your website, as well as
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:5]: When I read this, not sure about which price we are talking about

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+condition-specific prices based on selected criteria.
+
+Tax display strategy
+====================
+
+The choice of tax display usually depends on a country's regulations or the type of customers **(B2B
+vs B2C)**. To select the type of price displayed, go to :menuselection:`Website --> Configuration
+--> Settings`, scroll-down to the :guilabel:`Pricing` category, and select :guilabel:`Tax-Excluded`
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:13]:
```suggestion
--> Settings`, scroll down to the :guilabel:`Pricing` category, and select :guilabel:`Tax-Excluded`
```

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+Pricing management
+==================
+
+Odoo offers multiple options to select the price displayed on your website, as well as
+condition-specific prices based on selected criteria.
+
+Tax display strategy
+====================
+
+The choice of tax display usually depends on a country's regulations or the type of customers **(B2B
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:11]:
```suggestion
The choice of tax display usually depends on the country's regulations or the type of customers **(B2B
```

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+vs B2C)**. To select the type of price displayed, go to :menuselection:`Website --> Configuration
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:12]:
```suggestion
vs B2C)**. To select the type of price displayed, go to :menuselection:`Website app --> Configuration
```

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+Price management
+================
+
+There are multiple ways to manage the pricing of your products.
+
+Pricelists
+----------
+
+Pricelists are the main tool to manage prices on your ecommerce. They allow you to define specific
+prices based according to: **country groups**, **currency**, **minimum quantity**, or **variants**.
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:34]:
```suggestion
prices according to: **country groups**, **currency**, **minimum quantity**, or **variants**.
```

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+Pricelists can be selected by your customer on the product's page in the upper-right corner. You can
+create as many pricelist as needed, but it is mandatory to have at least one pricelist available on
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:36]:
```suggestion
create as many pricelists as needed, but it is mandatory to have at least one pricelist available on
```

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+a website. If no custom pricelists are added, Odoo defaults to the **Public Pricelist** for all
+websites.
+
+To create or configure a pricelist, go to :menuselection:`Website backend --> Products -->
+Pricelists`. You can change the :guilabel:`Name` of the pricelist, the :guilabel:`Currency`, the
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:41]:
```suggestion
Pricelists`. You can change the :guilabel:`Pricelist Name`, the :guilabel:`Currency`, the
```

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+:guilabel:`Company` in which it is available, and the :guilabel:`Price Rules`. When modifying or
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:42]:
```suggestion
:guilabel:`Company` in which it is available. When modifying or
```

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+creating a line under :guilabel:`Price Rules`, you can:
+
+.. note::
+   To offer **multiple currencies** to your customer, make sure to activate the
+   :guilabel:`Multi-Currencies` option in :menuselection:`Website backend --> Configuration -->
+   Settings` under the category :guilabel:`Pricing`.
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:48]:
```suggestion
Activate the :guilabel:`Multi-Currencies` option to offer a multi-currencies environment to your customers. Go to the :menuselection:`Website backend ‣ Configuration ‣ Settings` and under the :guilabel:`Pricing` category check the :guilabel:`Multi-Currencies` box.
```

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+
+- :guilabel:`Apply On`: choose to what type the pricelist applies (all products, product category,
+  single product, or product variant(s));
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:51]:
```suggestion
- :guilabel:`Apply On`: choose to what type the pricelist applies (All Products, Product Category,
  Product, or Product Variant;
```

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+- :guilabel:`Compute Price`: how the price should be computed (fixed price, percentage as discount,
+  or a formula of your choice);
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:53]:
```suggestion
- :guilabel:`Compute Price`: how the price should be computed (Fixed Price, Percentage discount,
  or Formula)
```

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 12]
```diff
@@ -0,0 +1,185 @@
...
+- :guilabel:`Min. Quantity`: the minimum quantity at which the :guilabel:`price` is applicable;
+- :guilabel:`Fixed` or :guilabel:`Percentage Price`: the fixed price of the product (or variant) or
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:55]: I don't see the Percentage price option

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 13]
```diff
@@ -0,0 +1,185 @@
...
+  the percentage price;
+- :guilabel:`Start` and :guilabel:`End Date`: the dates during which the pricelist is available.
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:57]:
```suggestion
- :guilabel:`Start Date` and :guilabel:`End Date`: the dates during which the pricelist is available.
```

File: content/applications/websites/ecommerce/getting_started/pricing_management.rst [hunk 14]
```diff
@@ -0,0 +1,185 @@
...
+
+.. image:: pricing_management/pricing-configuration.png
+   :align: center
+   :alt: Pricelist configuration
+
+Add multiple currencies
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can assign a specific currency to a pricelist and therefore offer **multiple currencies** to
+your customers on the ecommerce. In order for your customers to be able to switch currencies, make
```
Comment (Donapi) [inline content/applications/websites/ecommerce/getting_started/pricing_management.rst:81]:
```suggestion
your customers on the eCommerce. In order for your customers to be able to switch currencies, make
```

**Comment by toaa-odoo** [comment]
Closing as it will be down-ported from 16.

**Pull request closed without merging**